metabase/metabase
# Update obsolete repro for #12985

### Status
READY

### What does this PR accomplish?
- updates obsolete (still skipped) repro for #12985 

### Additional notes:
- this test was referencing `.Icon-funnel_add` which was recently replaced with `.Icon-filter` in the dashboard (it's also not in use anywhere else in the project anymore)
- this PR updates that icon reference and
- improves the test in the areas in which it was failing locally

### How to test this manually/locally?
_Because the related issue is still not fixed, it is possible to test this only locally and observe the test failing at the very last step. It should work before that._
- `yarn test-cypress-open`
- `frontend/test/metabase/scenarios/question/filter.cy.spec.js:12`
- replace `it.skip()` with `it.only()`
